neurodamus/connection_manager.py

```diff
@@ -1128,12 +1124,12 @@ def finalize(self, base_seed=0, sim_corenrn=False, *, _conn_type="synapses", **c
         n_created_conns = 0
 
         for popid, pop in self._populations.items():
-            attach_src = pop.src_id == 0 or not pop.virtual_source  # real populations
+            attach_src = pop.src_pop_id == 0 or not pop.virtual_source  # real populations
```

Comment:
```suggestion
            attach_src = not pop.virtual_source  # real populations
```
I think we can simply here. Only read populations requires the netcon to attach to src cell.

Reply:
Sure, I just let it there, cause looks more related with default population that we agreed to remove in a later PR right?

Reply:
Ok then, just don't forget that 👍